### DISCUSSION TITLE
Is there an RSS feed for the release notes?

### DISCUSSION THREAD

**Discussion by auno** [zed-industries/zed#16475] [Feature Requests]
It would be great to be able to keep up with the release notes in my normal news/reading flow, but, while I've found the feed for the blog, I haven't been able to find an RSS feed for the release notes. From the blog RSS link, I would guess at https://zed.dev/releases/stable.rss for the release notes, but that just gives me a 500 response. Maybe it's already supposed to be there but is just broken?

**Comment by bajrangCoder** [comment]
I don't think there is any rss or maybe something wrong on that url.

Btw you can follow zed social handle on X(twitter) for release notes or Follow zed's discord server, "Announcement" channel for detailed release notes

**Reply by auno** [reply]
That's too bad if that's the case. I don't use either Twitter or Discourse (or any other social media), so those aren't really alternatives for me.

**Reply by bajrangCoder** [reply]
I'm not entirely sure about that. However, you can use GitHub's "Watch" option to follow the repository. GitHub will notify you of new releases directly in your github dashboard.
Alternatively, in the Zed app, it automatically checks for updates, applies them, and displays notifications with links to the release notes. It can be helpfull.

**Reply by auno** [reply]
I was hoping for an existing RSS feed specifically to avoid being torn out of current tasks (i.e. by notifications within GitHub or Zed itself) and instead integrate it with the rest of my reading flow (where everything is RSS). But if it's not there, it's not there. It should be simple enough to solve it on my own end with a cronjob.

**Comment by AlpSha** [comment]
I was looking for the same thing. Ended up using Github's functionality. I'm sharing it in case anyone needs it.
https://github.com/zed-industries/zed/releases.atom

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links

**Comment by JosephTLyons** [comment]
Reopening this Discussion since it's a feature request and not a Q/A post. Seems like a reasonable ask. Ill look into it.

**Comment by JosephTLyons** [comment]
I've added a stable releases feed here:

https://zed.dev/stable-releases.rss
